third-party/realdds/src/dds-device-server.cpp

```diff
@@ -117,10 +129,22 @@ static void on_discovery_stream_header( std::shared_ptr< dds_stream_server > con
 
     topics::flexible_msg notification( msg );
     notifications.add_discovery_notification( std::move( notification ) );
+
+    auto opt_json = nlohmann::json();
```

Comment:
`stream_options`

Reply:
Let's be specific so it's clear, and use `json::array()`

Reply:
Using `stream_options` and `array`